services/api/package.json

```diff
@@ -49,7 +51,7 @@
     "juice": "^7.0.0",
     "mongodb": "^3.6.2",
     "nodemon": "^2.0.4",
-    "pino-pretty": "^4.3.0",
+    "pino-pretty": "^4.7.1",
```

Comment:
can remove this

Reply:
yeah pino is pretty weird actually... just using the `prettyPrint` key requires the `pino-pretty` package to be installed, however it's not a dependency of `pino`, instead you get

```
Missing `pino-pretty` module: `pino-pretty` must be installed separately
```

so I think we can't remove it... I tried to get where I needed with just the JSON config alone but it's just too static to work with, most things can't be changed